docs/reference-guides/rancher-webhook.md

```diff
@@ -11,12 +11,47 @@ Rancher-Webhook is an essential component of Rancher that works in conjunction w
 Rancher deploys Rancher-Webhook as a separate deployment and service in both local and downstream clusters. Rancher manages Rancher-Webhook using Helm. It's important to note that Rancher may override modifications made by users to the Helm release.
 
-## Why do we need it?
+Each Rancher version is designed to be compatible with a single version of the webhook. The compatible versions are provided below for convenience.
+
+**Note:** Rancher manages deployment and upgrade of the webhook. Under most circumstances, no user intervention should be needed to ensure that the webhook version is compatible with the version of Rancher that you are running.
+
+| Rancher Version | Webhook Version |
+|-----------------|:---------------:|
+| v2.7.0          |     v0.3.0      |
+| v2.7.1          |     v0.3.0      |
+| v2.7.2          |     v0.3.2      |
+| v2.7.3          |     v0.3.3      |
+| v2.7.4          |     v0.3.4      |
+| v2.7.5          |     v0.3.5      |
+| v2.7.6          |     v0.3.5      |
+
+
+## Why Do We Need It?
 
 Rancher-Webhook is crucial for Rancher to protect clusters against malicious attacks and enable various features.
 Rancher relies on the Rancher-Webhook as an integral part of its functionality. Without the webhook, Rancher would not be a complete product. 
 It provides essential protection for Rancher-managed clusters, preventing security vulnerabilities and ensuring the consistency and stability of the cluster.
 
+## What Resources Does the Webhook Validate?
+
+An in-progress list of the resources that the webhook validates can be found in the [webhook's repo](https://github.com/rancher/webhook/blob/release/v0.4/docs.md). Checks specific to one version can be found by viewing the `docs.md` file associated with a particular tag (note that webhook versions prior to `v0.3.6` won't have this file).
+
+## Bypassing the Webhook
+
+Sometimes, it may be necessary to bypass Rancher's webhook validation to perform emergency restore operations, or fix other critical issues. The bypass operation is exhaustive, meaning that no webhook checks will apply when this is used. It is not possible to bypass some checks and have others still apply - they are either all bypassed, or all active. 
+
+:::danger
+
+Rancher's webhook provides critical security protections. Bypassing the webhook should only be done by administrators in specific scenarios, after all other options have been exhausted. In addition, permission to bypass the webhook should be carefully controlled, and never given to users who are not admins.
+
+:::
+
+To bypass the webhook, impersonate both the `rancher-webhook-sudo` service account and the `system:masters` group (both are required):
+
+```bash
+kubectl create -f example.yaml --as=system:serviceaccount:cattle-system:rancher-webhook-sudo --as-group=system:masters
+```
+
```

Comment:
Update my typo below of `rancher-chart/rancher-webhook` -> `rancher-charts/rancher-webhook`

Reply:
I'd rather keep the scope of this doc narrow, particularly since those instructions need more details to work (even with the typo the instructions don't work since helm can't function due to the webhook blocking secrets access). That being said, I can give it a look.

Reply:
I've attempted to update this with more instructions, lmk if you have any further comments. I'll resolve this in favor of comments on the new diff.